web/vtadmin/src/hooks/api.ts

```diff
@@ -311,6 +317,42 @@ export const useSchema = (params: FetchSchemaParams, options?: UseQueryOptions<p
     });
 };
 
+/**
+ * useValidateKeyspace is a query hook that validates that all nodes reachable from the specified keyspace are consistent.
+ */
+export const useValidateKeyspace = (
```

Comment:
Thank you for adding types to these mutations ❤️ I know that it can be kinda tedious. This sets such a great example + I appreciate it!

Reply:
Np!! Thanks for holding the bar high!!